docs/notes-KZG10.md

```diff
@@ -1,3 +1,74 @@
 In this module we show how and why the KZG10 
-polynomial commitment scheme has been modularised 
-for this PLONK implementation.
\ No newline at end of file
+polynomial commitment scheme has been implemented
+for this PLONK implementation.
+
+KZG10 Commitments 
+==================
+
+PLONK can be constructed with different 
+commitment schemes and does not requre solely
+homomorphic commitments. However, this library
+has only homomorphic commitment schemes as they 
+are intelligble for users and have many useful
+properties. 
+
+In addition to the code complexity issue, 
+described immediately above, there is  
+another reason for choosing a commitment
+scheme with homomorphism. PLONK makes use
+of the linearisation technique, originally
+concieved in the SONIC [paper][sonic_paper]. 
+This technique requires the the commitment 
+scheme to be homomorphic. Without modification 
+of PLONK by means of remving the lineariser, 
+we are unable to use techniques such as 
+[FRI][fri_paper] protocol. 
+
+
+We use 'KZG10 commitments', often called 'Kate' 
+commitments refers to the commitments scheme 
+created by Kate, Zaverucha and Goldberg. 
+A deep explanation on how this particular commitment 
+scheme operates can be found [here][kzg10_paper] 
+in the original paper.
+There a multiple benefits of the KZG10 commitments
+aside from having readable code. It allow us to have 
+constant size commitments, which is achieved by having
```

Comment:
"It allow us to have constant size commitments"

When you say "constant size commitments" I think you need to say what it is in respects to?

"which is achieved by having the commitments as single elements."

I'm not sure this makes sense. I read it as "We can have constant size commitments by having the commitments as single elements"